project/Shared.scala

```diff
@@ -114,7 +114,8 @@ object Shared {
 
   lazy val tachyonSettings: Seq[Def.Setting[_]] = {
     def tachyonVersion(sv: String) =
-      sv.takeWhile(_ != '-' /*get rid of -SNAPSHOT, -RC or whatever*/)
+      sv.dropWhile(!_.isDigit) /*get rid of the v in v1.6.0 for instance */
```

Comment:
ooops I'm wrong this actually needed to support weird versions like `v1.6.0-preview2`.

So this PR is good to go!